index.js

```diff
@@ -55,26 +53,16 @@ async function accessSpreadsheet(status, url) {
       'max-col': sheet.colCount,
       'return-empty': true,
   })
+  
+  const index = cells.findIndex(cell => cell.value === url);
```

Comment:
No need for a loop with a mutating variable, we can use the built-in `array.findIndex` method instead to find the index of the cell with the URL, which is much clearer and it avoids mutating any global variables. :tada: